packages/shared/graphql/types.ts

```diff
@@ -230,12 +230,22 @@ export type PlaywrightTestSources = {
   [fileName: string]: string;
 };
 
+export type PlaywrightTestStacks = {
+  [fileName: string]: {
+    file: string;
+    line: number;
+    column: number;
+    function?: string;
```

Comment:
"function" is a reserved keyword. Can we use `functionName` (also this would be more consistent with other related types)

Reply:
Sadly, this is coming from Playwright as `function` but we can adapt it in the plugin